.github/workflows/ci.yml

```
@@ -0,0 +1,45 @@
+        run: |
+          python -m pip install --upgrade pip
+          pip install ruff pytest
+          pip install -e .
+      - name: Backtest pipeline
+        run: |
+          python - <<'PY'
+          from src import data_prep, strategy
+          assert data_prep.load_data("tests/fixtures/EURUSD_M15_sample.csv")
+          assert strategy.generate_signals() == []
+          print("stub back-test OK")
+          PY
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)